recipes/megagta/meta.yaml

```diff
@@ -0,0 +1,27 @@
+package:
+  name: megagta
+  version: "0.1_alpha"
+source:
+  git_url: https://github.com/HKU-BAL/megagta.git
+  git_rev: v0.1-alpha
+requirements:
+  build:
+    - automake
+    - autoconf
+    - m4
+    - gcc
```

Comment:
please use gcc  # [linux] and llvm # [osx] as build deps and libgcc # [linux] as run time dep